src/lib/controllers/ui-controller.js

```diff
@@ -651,68 +680,91 @@ export default class UIController {
 
     // Set initial values of components
     this.setRootComponentClasses()
-  }
 
-  static get defaults () {
-    return {
-      irregularBaseFontSizeClassName: 'alpheios-irregular-base-font-size'
-    }
+    this.state.activateUI()
+    document.body.dispatchEvent(new Event('Alpheios_Options_Loaded'))
+
+    const currentLanguageID = LanguageModelFactory.getLanguageIdFromCode(this.options.items.preferredLanguage.currentValue)
+    this.options.items.lookupLangOverride.setValue(false)
+    this.updateLanguage(currentLanguageID)
+    this.updateVerboseMode()
+    this.updateLemmaTranslations()
+    this.notifyInflectionBrowser()
+
+    this.state.setWatcher('uiActive', this.updateAnnotations.bind(this))
+
+    // Activate listeners
+    MouseDblClick.listen('body', evt => this.getSelectedText(evt))
+    document.addEventListener('keydown', this.handleEscapeKey.bind(this))
+    document.body.addEventListener('Alpheios_Reload', this.handleReload.bind(this))
```

Comment:
I think we need to move it to webextension. In Safari (I think, but can be wrong) if a new version of extension is available, a containing application needs to be run once. It will replace a native macOS background extension code with the new one. But the page will run an old version of a script still. But if we reload a page manually, it will pick up an updated content script version. Something along those lines.